services/identity/src/main/java/com/crapi/config/JwtProvider.java

```diff
@@ -142,6 +143,8 @@ private String getJwtSecret(JWSHeader header) throws JOSEException {
    */
   public boolean validateJwtToken(String authToken) {
     try {
+      if (JWTParser.parse(authToken) instanceof PlainJWT)
```

Comment:
Put this in the catch section as a fallback

Reply:
Done